### PR TITLE
20759582: Featured Job Card Component

### PR DESCRIPTION
**Pull request by tinkabel85**

### CODE REVIEW

File: src/components/components/FeaturedJobCard/FeaturedJobCard.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./FeaturedJobCard.scss";
+
+function FeaturedJobCard({ job, logoNameToPath }) {
+  return (
+    <>
```
Comment (jasper-prgh) [inline src/components/components/FeaturedJobCard/FeaturedJobCard.js]: You dont actually need the Fragment (<></>) here

File: src/components/components/FeaturedJobCard/FeaturedJobCard.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      line-height: 1.3em;
+    }
+  }
+  
+  &__info {
+    &__title {
+      margin: 0;
+      color: var(--color-bunting);
+      line-height: 1.3em;
+    }
```
Comment (jasper-prgh) [inline src/components/components/FeaturedJobCard/FeaturedJobCard.scss:37]: add an empty line here please

File: src/components/components/FeaturedJobCard/FeaturedJobCard.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    &__working-time {
+      margin: 0;
+      color: var(--color-text);
+    }
+  }
```
Comment (jasper-prgh) [inline src/components/components/FeaturedJobCard/FeaturedJobCard.scss:43]: formatting

**Review by jasper-prgh — DISMISSED** [review]

**Review by amindu00 — COMMENTED** [review]
can you add state models like in Candidate cards if possible

**Comment by tinkabel85** [comment]
@Star4ze , I have removed everything from the Card (both CandidateProfileCard & FeaturedJobCard) because I thought it belongs to their "parent" components (like Meet Candidates or Features Job Section Components)
e.g. would be smth like: 

        {staticCandidateCards.map((candidate) => (
          <CandidateProfileCard candidate={candidate} />
        ))}

File: src/components/components/FeaturedJobCard/FeaturedJobCard.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "./FeaturedJobCard.scss";
+
+function FeaturedJobCard({ job, logoNameToPath }) {
```
Comment (timmy38233) [inline src/components/components/FeaturedJobCard/FeaturedJobCard.js]: You won't need LogoNameToPath here. Just access the logo like ``, the parent component will handle the rest :)

File: src/components/components/FeaturedJobCard/FeaturedJobCard.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  return (
+      <div className="FeaturedJobCard">
+        <div className="FeaturedJobCard__header">
+          <img
+            className="FeaturedJobCard__header__img"
+            src={logoNameToPath}
+            alt="Company Logo"
```
Comment (timmy38233) [inline src/components/components/FeaturedJobCard/FeaturedJobCard.js]: As an alt-Text you can use job.logoName or job.company

File: src/components/components/FeaturedJobCard/FeaturedJobCard.scss [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 27px 24px;
+  gap: 1em;
+  border-radius: 1em;
+  width: 25em;
+  height: 20.5em;
+  background-color: var(--color-white);
+  border: 1px dashed red;
```
Comment (timmy38233) [inline src/components/components/FeaturedJobCard/FeaturedJobCard.scss]: I don't think we need a red border here, do we?

**Review by timmy38233 — CHANGES_REQUESTED** [review]
Code looks great overall! :)
Will check in-browser-appearance after the small changes

File: src/components/components/FeaturedJobCard/FeaturedJobCard.js (hunk 4 above)
Comment (tinkabel85) [reply src/components/components/FeaturedJobCard/FeaturedJobCard.js]: done! thnx)

File: src/components/components/FeaturedJobCard/FeaturedJobCard.js (hunk 5 above)
Comment (tinkabel85) [reply src/components/components/FeaturedJobCard/FeaturedJobCard.js]: done!

File: src/components/components/FeaturedJobCard/FeaturedJobCard.scss (hunk 6 above)
Comment (tinkabel85) [reply src/components/components/FeaturedJobCard/FeaturedJobCard.scss]: sure, I forgot to remove))
I usually add it for testing purposes - to see all elements on browser

**Review by timmy38233 — APPROVED** [review]
Looks great! :)

**Review by Puxi19 — APPROVED** [review]

**Review by amindu00 — APPROVED** [review]

**Pull request merged**